_includes/video

```diff
@@ -1,5 +1,6 @@
 {% capture video_id %}{{ include.id }}{% endcapture %}
 {% capture video_provider %}{{ include.provider }}{% endcapture %}
+{% capture video_danmaku %}{{ include.danmaku }}{% endcapture %}
```

Comment:
I'm afraid this isn't going to work as expected. With `{% capture %}` you're turning this variable into a string, which is always truthy in Ruby. As you're applying `default` below, you'd better use `{% assign %}` so null values are preserved.

```diff
-{% capture video_danmaku %}{{ include.danmaku }}{% endcapture %}
+{% assign video_danmaku = include.danmaku %}
```